src/commcare_cloud/fab/operations/release.py

```diff
@@ -229,7 +228,7 @@ def record_successful_release():
         files.append(RELEASE_RECORD, str(env.code_root), use_sudo=True)
 
 
-#TODO make this a nicer task
+# TODO make this a nicer task
```

Comment:
This TODO dates back to 2016, written by a dev who is no longer at Dimagi. I think we should either create a ticket and put it in the next sprint ... or ... preferably ... just delete the TODO cos it ain't gonna happen. :upside_down_face:

Reply:
I guess if it dates back to 2016 it's not needed, so I'd rather remove the comment.